components/datepicker/DatePickerRange.js

```diff
@@ -0,0 +1,107 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import DayPicker from 'react-day-picker';
+import NavigationBar from './NavigationBar';
+import WeekDay from './WeekDay';
+import { convertModifiersToClassnames, hasRange, isSelectingFirstDay } from './utils';
+import cx from 'classnames';
+import theme from './theme.css';
+
+class DatePickerRange extends PureComponent {
+  state = {
+    from: null,
+    to: null,
+    enteredTo: null,
```

Comment:
I don't understand the `to` and `enteredTo` stuff. It's not clear what it actually means.

Reply:
+1

Reply:
I renamed these props to something more explicit.